server/service/src/processors/general_processor.rs

```diff
@@ -91,12 +99,13 @@ pub(crate) fn process_records(
 
 pub(super) trait Processor {
     fn get_description(&self) -> String;
+
     /// Default to using change_log_table_names
-    fn changelogs_filter(&self) -> ChangelogFilter {
-        ChangelogFilter::new().table_name(EqualFilter {
+    fn changelogs_filter(&self, _ctx: &ServiceContext) -> Result<ChangelogFilter, ProcessorError> {
```

Comment:
Altered this method to accept context, so can query for active stores on site in changelog filter

Reply:
Nice, this is good extension, vs checking during processing